backend/compress/compress.go

```diff
@@ -418,7 +418,8 @@ func (f *Fs) verifyObjectHash(ctx context.Context, o fs.Object, hasher *hash.Mul
 		if err != nil {
 			fs.Errorf(o, "Failed to remove corrupted object: %v", err)
 		}
-		return errors.Errorf("corrupted on transfer: %v compressed hashes differ %q vs %q", ht, srcHash, dstHash)
+		//return errors.Errorf("corrupted on transfer: %v compressed hashes differ %q vs %q", ht, srcHash, dstHash)
+		return errors.Errorf("%v corrupted on transfer: source %q, destination %q", ht, srcHash, dstHash)
```

Comment:
This message is probably strange:
"md5 corrupted on transfer: ..."
How can md5 be corrupted?

Reply:
### Testing everything
**I** _wonder_ 
> How can md5 be corrupted?

 ` too` 
@KiwiAp 
#1 
[https://github.com/rclone/rclone/issues/5268](url)

Reply:
> This message is probably misleading:
> "md5 corrupted on transfer: ..."
> How can md5 be corrupted?

Let's see what will happen if I quote your reply.